mappings/net/minecraft/structure/AdvancedStructureStart.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_4183 net/minecraft/structure/AdvancedStructureStart
```

Comment:
Is it really "advanced" though? The only difference seems to be that it has a margin and will include more adjacent chunks in world-generation.